distributed/comm/tests/test_ucx.py

```diff
@@ -36,12 +41,13 @@ def test_registered():
         await q.put(comm)
 
     listener = listen(listen_addr, handle_comm, connection_args=listen_args, **kwargs)
-    with listener:
-        comm = await connect(
-            listener.contact_address, connection_args=connect_args, **kwargs
-        )
-        serv_com = await q.get()
-        return comm, serv_com
+    await listener.start()
```

Comment:
You probably want `async with listener` here instead

Reply:
Thanks!  Resolved